src/main/java/longtimenosee/model/policy/Title.java

```diff
@@ -35,7 +35,7 @@ public Title(String title) {
      * Returns true if a given string is a valid name.
      */
     public static boolean isValidTitle(String test) {
-        return test.matches(VALIDATION_REGEX);
+        return test.matches(VALIDATION_REGEX) && test.length() <= 40;
```

Comment:
Small nitpick but perhaps instead of using 40 (which might be a magic number), we could define it as a constant?
i.e: `public static final int MAXIMUM_MESSAGE_LENGTH = 40` over at line 11/12